ios/Sources/AboutFeature/SelectedType.swift

```diff
@@ -0,0 +1,15 @@
+import Styleguide
+
+public enum SelectedType: Int, CaseIterable {
+    case staff = 0
+    case contributor = 1
+
+    var title: String {
```

Comment:
this calls only in aboutScreen. It seems better to use as private extension in `AboutScreen.swift`